Remove references to Notify team, in favour of Emergency Alerts (#4211)
alphagov/govuk-developer-docs: source/manual/emergency-alerts.html.md

---
owner_slack: "#emergency-alerts"
title: What are Emergency Alerts?
layout: manual_layout
type: learn
parent: "/manual.html"
section: Emergency Alerts
important: true
---

> The Cabinet Office and the Department for Digital, Culture, Media & Sport are developing a ‘cell broadcast’ alert system to enable people whose lives are at risk in an emergency to be rapidly contacted via their mobile phone. They are currently at the testing phase and subject to successful progress, hope to launch the service soon.

Reviewer: ```suggestion
The Cabinet Office and the Department for Digital, Culture, Media & Sport have developed a ‘cell broadcast’ alert system to enable people whose lives are at risk in an emergency to be rapidly contacted via their mobile phone. 
```

API Keys
--------

> Emergency Alerts have three Fastly API keys with [`purge_select` scope](https://developer.fastly.com/reference/api/auth/#scopes), one for

Reviewer: ```suggestion
The Emergency Alerts team have three Fastly API keys with [`purge_select` scope](https://developer.fastly.com/reference/api/auth/#scopes), one for
```

integration, staging and production. These allow them to purge individual pages or surrogate keys from the cache. Note that the scope

> of these keys is not restricted to `/alerts` - in principle they could be used to purge any page from the cache. We trust Emergency Alerts to take

Reviewer: ```suggestion
of these keys is not restricted to `/alerts` - in principle they could be used to purge any page from the cache. We trust the Emergency Alerts team to take
```
